bio2zarr/vcf2zarr/verification.py

```diff
@@ -114,7 +114,7 @@ def assert_format_val_equal(vcf_val, zarr_val, vcf_type, vcf_number):
     assert isinstance(vcf_val, np.ndarray)
     if vcf_type in ("String", "Character"):
         assert len(vcf_val) == len(zarr_val)
-        for v, z in zip(vcf_val, zarr_val):
+        for v, z in zip(vcf_val, zarr_val, strict=False):
```

Comment:
What's the motivation for the ``strict=False`` arguments here and elsewhere? In most of these cases I would have thought we'd want ``strict=True``, so adding the explicit False argument gives a false impression that we expect the iterators to be different lengths.

Reply:
Yes, this was automatically fixed by ruff, but you're right that we want `strict=True`. I've changed this here and in vcztools.